test/test_process.py

```diff
@@ -117,6 +118,28 @@ def test_removal_no_bootloader(self) -> None:
                              limit=1,
                              destructive=False)
 
+    def test_removal_bootloader_all_kernels(self) -> None:
+        td = Path(self.td.name)
+
+        class MockBootloader(Bootloader):
+            name = 'mock'
```

Comment:
Minor thing: could you please use `""` in new code? The package originally didn't strictly follow PEP 8 but I try to do that now.

Reply:
Fixed in this test only. I did not touch `SetUp()` in this file or anything in `process.py`. I was unsure if you had wanted me to change quotes in any other place.

Also, according to https://peps.python.org/pep-0008/#string-quotes string quotes are not strictly chosen as long as the project is consistent.